### PR TITLE
docs: reorganize installation chapter and related

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
This PR aims to cleanup our documentation and included some long awaited task.
It has serve as skeleton for numerous small changes, and as everything has been merged since, it can get retired now.

#### Please check

- [ ] Short description and the purpose of this PR is present _above this paragraph_
- [ ] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General
- [ ] PR name is meaningful
- [ ] Purpose of the PR is understood
- [ ] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [ ] Commit descriptions are understandable and well formatted
- [ ] If backport: add original PR number and target branch at top of this file: **Backport of PR#000 to bareos-2x**

##### Source code quality

- [ ] Source code changes are understandable
- [ ] Variable and function names are meaningful
- [ ] Code comments are correct (logically and spelling)
- [ ] Required documentation changes are present and part of the PR
- [ ] `bareos-check-sources --since-merge` does not report any problems
- [ ] `git status` should not report modifications in the source tree after building and testing

##### Tests

- [ ] Decision taken that a test is required (if not, then remove this paragraph)
- [ ] The choice of the type of test (unit test or systemtest) is reasonable
- [ ] Testname matches exactly what is being tested
- [ ] On a fail, output of the test leads quickly to the origin of the fault

### CODE REVIEW

**Comment by bruno-at-bareos** [comment]
@joergsteffens I would like to have your thoughts on the changes made here, especially about the still present table located at
[OperatingSystems](https://download.bareos.org/bareos/experimental/CD/PR-1203/BareosMainReference/Appendix/OperatingSystems.html#)

**Review by joergsteffens — COMMENTED** [review]
I left some initial remarks. If you which, I can also do a full review. However, I think it makes sense to split this PR.

Additional remarks:
* Install on Univention Corporate Server
  * Special Univention Corporate Server are no longer provided. Instead Debian packages should be used. Therefore this chapter should get replaced by a small hint, explaining this.
* I see that you updated the Debian install instructions, as required for newer versions of Debian. However it now even gets more complicated. I think we can make this a bit easier by providing more config files in our repository. To test this, it would make sense to split this in a seperate PR.
* I think the installation chapter should distinguish between server and client only installations. Maybe strictly splitting them in different chapters??? We can discuss this.
* While it is good that you added information for bareos.com customers, I think they still got it harder to find the required information. Maybe we find a way to make installation easier for them. Extra chapter for installing bareos.com? Of course, this would be partly redundant.

File: docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst [hunk 1]
```diff
@@ -75,31 +79,40 @@ Bareos :sinceVersion:`15.2.0: requires: jansson` requires the Jansson library pa
...
+   # BAREOS_PASSWORD=Super#Encrypted/Password
+
    # See https://download.bareos.org/bareos/release/
+   # or https://download.bareos.com/bareos/release/
    # for applicable releases and distributions
 
-   DIST=EL_8
+   DIST=EL_9
    # or
+   # DIST=RHEL_9
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst:100]: Fixed, also there was several occurrence of EL_9.

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 2]
```diff
@@ -23,6 +23,9 @@ Additionally, the following table gives an overview on the available package ver
    RHEL,     current, v12.4, v12.4, v12.4
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply docs/manuals/source/Appendix/OperatingSystems.rst]: Updated

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 3]
```diff
@@ -23,6 +23,9 @@ Additionally, the following table gives an overview on the available package ver
...
    SLES,     current, v12.4, v12.4, v12.4
    Ubuntu,   current, v12.4, v12.4, v12.4
+   Universal Linux Client,  openSSL 1.1, v21.0, v21.0, v21.0
+
+
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply docs/manuals/source/Appendix/OperatingSystems.rst]: Updated

File: docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst [hunk 4]
```diff
@@ -36,7 +36,8 @@ This will start a very basic Bareos installation which will regularly backup a d
 Decide about the Bareos release to use
 --------------------------------------
 
-You’ll find Bareos binary package repositories at https://download.bareos.org/bareos/. The stable releases are available at https://download.bareos.org/bareos/release/.
+You’ll find Bareos stable release binaries packages repositories at https://download.bareos.org/bareos/release.
+For those who have a valid subscription contract, the repositories are located at https://download.bareos.org/bareos/release/.
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst]: updated

**Review by joergsteffens — CHANGES_REQUESTED** [review]
Besides of this we still got the topics:

- Keeping the "Installing Bareos" section separate between Bareos server and client only installations.
- Univention Corporate Server no longer with special integration. Section can be replaced with a hint about this and a hint about how to use UCS5
- Handling of Debian installations

File: CHANGELOG.md [hunk 5]
```diff
@@ -148,6 +150,10 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - add instruction to run sphinx linkcheck before release [PR #1218]
 - FreeBSD install create /usr/local/etc/pkg/repos directory [PR #1227]
 - remove deprecated Rados & ceph references from documentation [PR #1216]
+- dir-job-RescheduleTimes remove wrong default sentence [PR #1203]
+- FreeBSD install create /usr/local/etc/pkg/repos directory [PR #1203]
+- update Appendix/HardwareSizing DB size with new numbers and formulas [BUG #1477][PR #1203]
```
Comment (bruno-at-bareos) [inline CHANGELOG.md:155]: Duplicate of PR #1231
```suggestion

```

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 6]
```diff
@@ -20,24 +20,26 @@ Additionally, the following table gives an overview on the available package ver
...
    :ref:`Univention Corporate Linux <section-UniventionCorporateServer>`, App Center, v12.4, v12.4, v12.4
 
    :strong:`MS Windows`
    :ref:`MS Windows <section-windows>` 32bit, 10/8/7, v12.4, v15.2, v15.2
-                                           , 2008/Vista/2003/XP, v12.4–v14.2
-   :ref:`MS Windows <section-windows>` 64bit, 10/8/2012/7, v12.4, v15.2, v15.2
-                                            , 2008/Vista, v12.4–v14.2
+                                           , 2008/8/2003/XP, v12.4–v14.2
+   :ref:`MS Windows <section-windows>` 64bit, 11/2022/2019/2016/10, v12.4, v15.2, v15.2
+                                            , 8/2012/7/2008/Vista, v12.4–v14.2
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply docs/manuals/source/Appendix/OperatingSystems.rst:35]: I think the >=7 is the best solution, until Microsoft change something in VSS API.

File: docs/manuals/source/Appendix/OperatingSystems.rst [hunk 7]
```diff
@@ -20,24 +20,26 @@ Additionally, the following table gives an overview on the available package ver
...
```
Comment (bruno-at-bareos) [inline docs/manuals/source/Appendix/OperatingSystems.rst:34]:
```suggestion
                                           , 2008/Vista/2003/XP, v12.4–v14.2
   :ref:`MS Windows <section-windows>` 64bit, >= 7, v12.4, v15.2, v15.2
                                            , 2008/Vista, v12.4–v14.2
```

**Comment by bruno-at-bareos** [comment]
- Did you have example or suggestion for this topic "Univention Corporate Server no longer with special integration. Section can be replaced with a hint about this and a hint about how to use UCS5" ?
- For Debian once we have the internal new helper ready the section can be simplified
- About the split this is on the next dev meeting agenda

File: CHANGELOG.md [hunk 8]
```diff
@@ -115,6 +115,8 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - console: aborting job run if jobid doesn't exist in catalog [PR #1188]
 - daemons: changed daemon CLI arguments parsing [PR #1187]
 - config parser: Refactor config reloading by using shared pointers instead of callbacks [PR #1151]
+- daemons: changed daemon CLI arguments parsing [PR #1187]
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply CHANGELOG.md]: done in latest changes of CHANGELOG.md

**Comment by bruno-at-bareos** [comment]
Waiting PR #1238 to be merged before restructuring further.

**Comment by bruno-at-bareos** [comment]
The only changes missing (deleting docs/manuals/source/include/images/dbconfig-2-select-database-type.png) will be operated into a clean new PR.

**Pull request closed without merging**